lang/python/numpy/patches/003-without-vendored-meson.patch

```diff
@@ -0,0 +1,22 @@
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -1,6 +1,6 @@
+ [build-system]
+-build-backend = "mesonpy"
+-backend-path = ['./vendored-meson/meson-python']
```

Comment:
any hint as to why we need to comment out this (and the code)?
i'm assuming that this helps us use the build system of openwrt somehow; though i'm somewhat vague here

Reply:
maybe you should try building this package without this patch. it will explain to you what happens in the absence of this patch. I also saw this issue reported in numpy but I forgot to link about this issue.